src/Presentation/Nop.Web/Administration/Views/Shared/Menu.cshtml

```diff
@@ -26,7 +26,7 @@
     if (!String.IsNullOrEmpty(url))
     {
         <li @(isActive ? Html.Raw("class=\"active current-active-item\"") : null)>
-            <a href="@url" class="menu-item-link">
+            <a href="@url" class="menu-item-link" @(item.OpenUrlInNewTab ? Html.Raw("target=\"_blank\"") : null)>

```

Comment:
It's better to remove space before @. And add it inside "Html.Raw". This way no empty space is generated with "OpenUrlInNewTab" set to "false"